pkg/controllers/organization/service_proxy_controller.go

```diff
@@ -106,3 +131,19 @@ func (r *ServiceProxyReconciler) reconcileServiceProxy(ctx context.Context, org
 	}
 	return nil
 }
+
+func (r *ServiceProxyReconciler) enqueueAllOrganizationsForServiceProxyPluginDefinition(ctx context.Context, o client.Object) []ctrl.Request {
+	return listOrganizationsAsReconcileRequests(ctx, r.Client)
+}
+
+func listOrganizationsAsReconcileRequests(ctx context.Context, c client.Client, listOpts ...client.ListOption) []ctrl.Request {
+	var organizationList = new(greenhousesapv1alpha1.OrganizationList)
+	if err := c.List(ctx, organizationList, listOpts...); err != nil {
+		return nil
+	}
+	res := make([]ctrl.Request, len(organizationList.Items))
+	for idx, organization := range organizationList.Items {
+		res[idx] = ctrl.Request{NamespacedName: client.ObjectKeyFromObject(organization.DeepCopy())}
```

Comment:
Why deepcopy? Request only needs name and namespace right?

Reply:
You're right. Fixed it.